src/main/java/Blossom.java

```diff
@@ -0,0 +1,90 @@
+import java.util.Scanner;
```

Comment:
Great job listing the classes used explicitly, instead of using the all (*) operator.